conf/bleedingEdge.neon

```diff
@@ -18,3 +18,6 @@ parameters:
 		phpDocParserRequireWhitespaceBeforeDescription: true
 		runtimeReflectionRules: true
 		notAnalysedTrait: true
+
+rules:
+	- PHPStan\Rules\Methods\MissingMagicSerializationMethodsRule
```

Comment:
This isn't how it's done, please check how other rules are enabled with bleedingEdge via conditionalTags and the parameters above. Thanks.